### PR TITLE
Add IMU sensor and message

### PR DESCRIPTION
**Pull request by corwinmacmillan**
Pending testing with adafruit sensor

### CODE REVIEW

File: catkin_ws/src/aquadrone_sensors/nodes/imu_node.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import rospy
+from aquadrone_msgs.msg import AquadroneIMU
+from sensor_msgs.msg import Imu
+from Adafruit_BNO005 import BNO005
+
+#print(dir(BNO005))
+bno = BNO005.BNO055()
+
+rospy.init_node('aquadrone_IMU')
+pubUW = rospy.Publisher('/aquadrone_v2/out/SensorUW', aquadroneIMU)
```
Comment (amaarquadri) [inline catkin_ws/src/aquadrone_sensors/nodes/imu_node.py]: What does 'UW' stand for?

File: catkin_ws/src/aquadrone_sensors/nodes/imu_node.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+pubIMU = rospy.Publisher('/aquadrone_v2/out/imu', Imu)
+rate = rospy.Rate(10)
+
+while not rospy.is_shutdown():
+  msgUW = aquadroneIMU()
```
Comment (amaarquadri) [inline catkin_ws/src/aquadrone_sensors/nodes/imu_node.py]: Shouldn't this be AquadroneIMU()

File: catkin_ws/src/aquadrone_sensors/nodes/imu_node.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+from sensor_msgs.msg import Imu
+from Adafruit_BNO005 import BNO005
+
+#print(dir(BNO005))
+bno = BNO005.BNO055()
+
+rospy.init_node('aquadrone_IMU')
+pubUW = rospy.Publisher('/aquadrone_v2/out/SensorUW', aquadroneIMU)
+pubIMU = rospy.Publisher('/aquadrone_v2/out/imu', Imu)
+rate = rospy.Rate(10)
```
Comment (amaarquadri) [inline catkin_ws/src/aquadrone_sensors/nodes/imu_node.py]: TODO: make this configurable

**Comment by Wesley-Fisher** [comment]
This is actually outdated now. Unfortunately, the code Corwin found for using the IMU did not work on the Jetson (it was made for the Pi, and is not fully compatible). I only just saw the button to close this here, so I'm closing this PR.

There is code used in the Jetson repo that works with the imu.

**Comment by amaarquadri** [comment]
Shouldn't we still have the code for the IMU node itself in this repo?

**Comment by Wesley-Fisher** [comment]
It's either part of the Aquadrone2020Jetson repo, or is a subrepo used by it. We can certainly addit to our main repo if needed, but the code will only ever be run on the Jetson. It may just add extra dependencies for others developing unrelated code.

**Pull request closed without merging**